spec/features/idv/outage_spec.rb

```diff
@@ -238,48 +238,66 @@ def sign_in_with_idv_required(user:, sms_or_totp: :sms)
     end
   end
 
-  %w[acuant lexisnexis_instant_verify lexisnexis_trueid].each do |service|
```

Comment:
this was nesting the examples, best viewed with [`?w=1`](https://github.com/18F/identity-idp/pull/9552/files?w=1)